src/features/diagnosticsProvider.ts

```diff
@@ -275,25 +283,22 @@ class DiagnosticsProvider extends AbstractSupport {
                 let value = await serverUtils.codeCheck(this._server, { FileName: null }, this._projectValidation.token);
 
                 let quickFixes = value.QuickFixes
-                    .filter(DiagnosticsProvider._shouldInclude)
                     .sort((a, b) => a.FileName.localeCompare(b.FileName));
 
                 let entries: [vscode.Uri, vscode.Diagnostic[]][] = [];
                 let lastEntry: [vscode.Uri, vscode.Diagnostic[]];
 
-                for (let quickFix of quickFixes) {
-
-                    let diag = DiagnosticsProvider._asDiagnostic(quickFix);
-                    let uri = vscode.Uri.file(quickFix.FileName);
+                for (let quickFix of this._mapQuickFixesAsDiagnostics(quickFixes)) {
```

Comment:
Sorry for nitpicking but this is confusing me a bit. We are calling mapQuickFixesAsDiagnostics then again calling it quickFix.

Reply:
> Sorry for nitpicking but this is confusing me a bit. We are calling mapQuickFixesAsDiagnostics then again calling it quickFix.

I don't think that was nitpicking, naming is important and that is definetly mistake by me 😄